Modular_PROPOSAL/swarm_bridge/bridge.py

```
@@ -0,0 +1,403 @@
+    
+    async def health(self) -> Dict[str, Any]:
+        """Report process health"""
+        try:
+            if self.process.is_running():
+                return {
+                    "status": "healthy",
+                    "pid": self.process.pid,
+                    "cpu": self.process.cpu_percent(),
+                    "memory": self.process.memory_info().rss / 1024 / 1024
+                }
+        except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)